docker-compose.yml

```diff
@@ -11,7 +11,7 @@
 version: '2.1'
 services:
   postgresql:
-    image: chefdemo/postgresql:stable
+    image: chefdemo/postgresql:${VERSION:-stable}
```

Comment:
I believe the omission of VERSION was intentional before, since the VERSION of postgresql and elasticsearch doesn't really match up the rest of the services.  Perhaps leave it off, or make it a variable name specific for postgresql?

I think we want to account for two scenarios: 
* running this compose with docker images build and published locally, from running `./habitat_pkgs_build.sh` (in which case VERSION won't be 'stable' and VERSION won't match what's local)
* running the compose to pull images from docker hub